### PR TITLE
Update backend_vars.asciidoc

### PR DESCRIPTION
**Pull request by srbala**
Simple typo fix. 

* First one for markdown table alignment
* Second for port listen, log below for sample

```log
[2023-01-11T15:59:33.346235Z] [info] cmdsrv: daemon reachable under http://*:20013/_kAZUTRYVbxZFm3E/
[2023-01-11T15:59:33.347694Z] [info] Listening at "http://0.0.0.0:20013"
Web application available at http://0.0.0.0:20013
```

### CODE REVIEW

File: doc/backend_vars.asciidoc
Comment (Martchus) [inline doc/backend_vars.asciidoc:146]: I think 20002 is actually correct, e.g. check the assignment in https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/Worker/Settings.pm. Note that the command server port is "+ 1" (see https://github.com/os-autoinst/os-autoinst/blob/master/OpenQA/Isotovideo/Runner.pm) but this is about `QEMUPORT` itself.

**Review by Martchus — CHANGES_REQUESTED** [review]

**Comment by mimi1vx** [comment]
please squash commits,

**Comment by Martchus** [comment]
We can also squash via GitHub's merge button. So it LGTM now. However, I won't approve "officially" so mergify won't merge it automatically.

**Pull request merged**
